### DISCUSSION TITLE
Could not get main file: standalone-apiman.xml

### DISCUSSION THREAD

**Discussion by cleuson-alves** [apiman/apiman#2595] [Help]
Hello, I'm following the installation of apiman using the official documentation, but it's giving an error, I've already researched, but I haven't been able to find out yet. I'm using a VM with CentOS7 installed.
Command executed:
[.....wildfly-23.0.2.Final]# ./bin/standalone.sh -c standalone-apiman.xml

Error:
java.lang.IllegalStateException: WFLYCTL0214: Could not get main file: standalone-apiman.xml. Specified files must be relative to the configuration dir: /root/apiman-3.1.1.Final/wildfly-23.0.2.Final/standalone/configuration

I don't know which parameter to use

**Comment by msavy** [comment]
I have never seen that error before.  Have you tried 3.1.2.Final?

Are you sure that `standalone-apiman.xml` is actually there? Error suggests the file is missing from your deployment.
